src/transformers/file_utils.py

```diff
@@ -119,6 +119,15 @@
 except ImportError:
     _has_apex = False
 
+
+try:
+    import faiss  # noqa: F401
+
+    _faiss_available = True
+except ImportError:
```

Comment:
Ok added it on successful import. Do you think it won't increse the logging a bit too much?  It will be shown each time someone imports `transformers`.

Reply:
I think Lysandre has now moved the default logging level to warning to not show these but now that you say it maybe it's only for the tests.

What about actually moving all these imports to `debug` (cc @LysandreJik @sgugger)

Reply:
Yes to that, I don't think it's needed at the info level (and please reduce the level to warning in datasets as well cause we get the logger.info from there when initializing transformers if both are installed ;-) )